plugins/microsoft_atp/komand_microsoft_atp/actions/update_alert/action.py

```diff
@@ -0,0 +1,25 @@
+import insightconnect_plugin_runtime
+from .schema import UpdateAlertInput, UpdateAlertOutput, Input, Output, Component
+
+# Custom imports below
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class UpdateAlert(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="update_alert", description=Component.DESCRIPTION, input=UpdateAlertInput(), output=UpdateAlertOutput()
+        )
+
+    def run(self, params={}):
+        # START INPUT BINDING - DO NOT REMOVE - ANY INPUTS BELOW WILL UPDATE WITH YOUR PLUGIN SPEC AFTER REGENERATION
+        alert_fields = params.get(Input.ALERT_FIELDS)
+        alert_id = params.get(Input.ALERT_ID)
+        # END INPUT BINDING - DO NOT REMOVE
+        self.logger.info("Running...")
+        self.logger.info(f"alert_id: {alert_id}")
+        self.logger.info(f"payload: {alert_fields}")
```

Comment:
Do you need to log that?

Reply:
Its not absolutely necessary, but I do find it helps when trying to see what alert id and fields are being used in the action.

Reply:
On this, if the information could be considered sensitive in any way I'd recommend you take it out. If the alert IDs and fields are completely insensitive then you can keep this in

Reply:
I went ahead and removed it since you can see it in the job details anyway.